.changeset/little-meals-thank.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/button': patch
```

Comment:
wasn't sure if this was a minor or a patch...

Reply:
We are adding a new feature (a new variant), so it is a minor